website/docs/d/search_indexes.html.markdown

```diff
@@ -30,17 +30,28 @@ data "mongodbatlas_search_index" "test" {
 
 ## Argument Reference
 
-* `project_id` - (Required) The ID of the organization or project you want to get the search index within.
-* `cluster_name` - (Required) The name of the cluster where you want to get the search index within.
+* `project_id` - (Required) Unique identifier for the [project](https://docs.atlas.mongodb.com/organizations-projects/#std-label-projects) that contains the specified cluster.
+* `cluster_name` - (Required) Name of the cluster containing the collection with one or more Atlas Search indexes.
 * `database_name` - (Required) Name of the database containing the collection with one or more Atlas Search indexes.
-* `collection_name` - (Required) Name of the collection with one or more Atlas Search indexes
+* `collection_name` - (Required) Name of the collection with one or more Atlas Search indexes.
 * `total_count` - Represents the total of the search indexes
 
 * `page_num` - Page number, starting with one, that Atlas returns of the total number of objects.
   
 * `items_per_page` - Number of items that Atlas returns per page, up to a maximum of 500.
 * `results` - A list where each represents a search index.
 
+### Results
```

Comment:
In the current documentation we've called this section Attribute Reference so to keep it consistent let's do the same here, e.g. https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/cloud_provider_snapshots#attributes-reference

Reply:
that makes sense, I already updated this part